e2e/test/buttons-tests/buttons-test.js

```diff
@@ -17,15 +17,16 @@ describe('FEF buttons tests', () => {
   it('Checks the the fef buttons page title and browser compare visual regression', () => {
     browser.url('/buttons');
     assert.equal(browser.getTitle(), 'Intellyo Application Design System');
-    assert(takeScreenshotAndGetWholePageCompareResult({defaultTolerance: 5.1, ignoreComparison: false, testDirPath: __dirname}), 'Whole FEF buttons page screenshot compare to a reference picture');
+    assert(takeScreenshotAndGetWholePageCompareResult({defaultTolerance: 5.1, ignoreComparison: false, testDirPath: __dirname}),
```

Comment:
`testDirPath` can be removed i guess

Reply:
This is removed in Feat/add select tests PR.